src/components/ViewArticles.jsx

```diff
@@ -1,9 +1,33 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
 function ViewArticles() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios.get('http://localhost:9090/api/articles')
+      .then(res => {
+        setArticles(res.data.articles);
+        console.log(res.data.articles);
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  }, []);
+
   return (
-    <div>ViewArticles</div>
-  )
+    <div>
+      <h1>View Articles</h1>
+      {articles.map(article => (
+    
+        <div key={article.article_id}>
```

Comment:
for the article previews we want to see a bit more information before we click on them and read the full article - like the number of votes and comments it has and the date it was posted (in a human readble format). We dont want to see the article body until we click through to a single article page

Reply:
Done!